operations/source_test.go

```diff
@@ -56,3 +56,20 @@ func TestCreateSourceWithPlatformType(t *testing.T) {
 	client.MustDo(exampleSource, createSource)
 	r.Equal(t, SourceID, exampleSource.ID)
 }
+
+func TestCreateSourceWithIP(t *testing.T) {
+	const (
+		SourceID = "src_test_5mygxph6d55vvy8nn9i"
+	)
+	client := testutil.NewFixedClient(t)
+
+	exampleSource, createSource := &omise.Source{}, &operations.CreateSource{
+		Type:     "wechat_pay",
+		Amount:   20000,
+		Currency: "thb",
+		Ip:       "192.168.1.1",
+	}
+
+	client.MustDo(exampleSource, createSource)
+	r.Equal(t, SourceID, exampleSource.ID)
```

Comment:
@aashishgurung Can we actually add a check for `IP` as this is the purpose of this test ?

Reply:
@AnasNaouchi That check will happen in `client.go`. I have added the check and test for it.

Reply:
Removed the code from `client.go` as we agreed on to not add check for it in our other libraries. The source test has been updated.